david_cpp.cc

```diff
@@ -1,9 +1,16 @@
 #include <iostream>
 using namespace std;
 
-int main(){
-	cout<<"Hello Martina!";
-	cout<<"Changes made here."<<endl;
-	cout<<"I am trying out the pull request stuff"<<endl;
+void print_Function()
+{
+	cout << "This is a print function" << endl;
+}
+
+int main()
+{
+	cout << "Hello Martina!";
+	cout << "Changes made here." << endl;
```

Comment:
Can you modify the print_Function so that it takes one parameter and prints it out?

Reply:
Sure!